### PR TITLE
Update sql-server-extended-security-updates.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

File: docs/sql-server/end-of-support/sql-server-extended-security-updates.md [hunk 1]
```diff
@@ -89,9 +89,7 @@ ESUs are available to customers running their workload in Azure, on-premises, or
...
 
-Azure services running [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] will receive ESUs automatically through existing [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] update channels, when the service is configured to use [automated patching](/azure/azure-sql/virtual-machines/windows/automated-patching).
-
-Azure services that have ***not* been configured for [automated patching](/azure/azure-sql/virtual-machines/windows/automated-patching)** will need to manually download and deploy ESU patches as described in the [on-premises or hosted environments](#on-premises-or-hosted-environments) section.
+Azure services running [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] will receive ESUs automatically through existing [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] update channels or Windows Update. Installation of IaaS extension is not needed to download SQL ESU Patches on Azure SQL VM's.
```
Comment (rwestMSFT) [inline docs/sql-server/end-of-support/sql-server-extended-security-updates.md:92]:
```suggestion
Azure services running [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] will receive ESUs automatically through existing [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] update channels or Windows Update. You don't need to install the SQL Server IaaS Agent extension to download ESU patches on an Azure SQL Virtual Machine.
```

**Comment by Jak-MS** [comment]
IMPORTANT: When this content is ready to merge, the Microsoft reviewer must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@Jak-MS If you're able to approve my suggested edit, we can get this merged.

**Comment by rwestMSFT** [comment]
#sign-off

**Pull request merged**